File: test/src/main/java/com/consultant/repository/UserRepository.java
package com.consultant.repository;

import com.consultant.domain.sys.User;
import org.spin.data.core.ARepository;
import org.springframework.stereotype.Repository;

/**
 * <p>Created by xuweinan on 2017/4/18.</p>
 *
 * @author xuweinan
 */
@Repository
public class UserRepository extends ARepository<User, Long> {
}

File: test/src/main/kotlin/com/shipping/repository/sys/UserRepository.kt
package com.shipping.repository.sys

import com.shipping.domain.sys.User
import org.spin.data.core.ARepository
import org.springframework.stereotype.Repository

/**
 *
 * Created by xuweinan on 2017/7/18.
 *
 * @author xuweinan
 */
@Repository
class UserRepository : ARepository<User, Long>()
